docs/UserGuide.md

```diff
@@ -262,49 +242,52 @@ Here are some fields that are shared amongst commands.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 
-Don't worry if your first few commands fail! PropertyWhiz's [output box](#navigating-the-user-interface) gives very detailed feedback on which parts of your input is invalid!
+Don't worry if your first few commands fail! PropertyWhiz's [output box](#navigating-the-user-interface) gives detailed feedback and you can try again.
 
 </div>
 
 ### Viewing help : `help`
 
-Shows a message explaining how to access the help page.
+Use this command if you need to access the help page.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
+<div style="page-break-after: always;"></div>
 
 ### Adding a property/buyer: `add`
 
-Adds a property/buyer to PropertyWhiz.
-
-* All other fields are compulsory other than tags.
-
-See the section [Valid properties/buyers](#valid-propertiesbuyers) for details on constraints on what you can enter.
+Use this command to add a property or buyer.
 
 Format:
 * Adding a property: `add property n/PROPERTY_NAME a/PROPERTY_ADDRESS s/SELLER_NAME p/SELLER_PHONE e/SELLER_EMAIL $/PRICE [t/TAG]…`
 * Adding a buyer: `add buyer n/BUYER_NAME p/BUYER_PHONE e/BUYER_EMAIL $/BUDGET [t/TAG]…`
 
 Examples:
-* `add property n/Blk 123 a/123, Clementi Rd, #04-20, 1234665 s/James Lee p/61234567 e/james@email.com $/100000 t/hdb t/3rm`
+* `add property n/Blk 123 a/123, Clementi Rd, #04-20, S654321 s/James Lee p/(hp) 61234567 e/james@email.com $/100000 t/hdb t/3rm`
 * `add buyer n/Sam p/91234567 e/sam@email.com $/740000 t/hdb t/3rm`
 
-Shown below is the output for the valid input `add property n/Blk 321 a/123, Clementi Rd, #04-20, 1234665 s/James Lee p/(hp) 61234567 e/james@email.com $/100000 t/hdb t/3rm`.
+Shown below is the output for the valid input `add property n/Blk 123 a/123, Clementi Rd, #04-20, S654321 s/James Lee p/(hp) 61234567 e/james@email.com $/100000 t/hdb t/3rm`.
```

Comment:
Just a minor detail, handphones dont start with 6 in singapore... Its fine if we dont change it.